src/main/resources/com/ibm/vCard/vCard_Format_2.1.xsd

```diff
@@ -75,11 +76,11 @@
        						 nilKind="literalValue" 
        						 nilValueDelimiterPolicy="none" 
        						 occursCountKind="implicit" 
-       						 outputNewLine="%CR;%LF;" 
+       						 outputNewLine="%LF;"
        						 representation="text" 
        						 separator="" 
        						 separatorPosition="infix" 
-       						 separatorSuppressionPolicy="anyEmpty" 
+       						 separatorSuppressionPolicy="trailingEmpty"
```

Comment:
Does this in fact matter? Does behavior change on IBM DFDL? We need this to remain portable. 
I realize this is more likely to affect unparsing, and hence, round-trip behavior.

Reply:
So using 'anyEmpty' is actually causing an error in a number of the tests.  There is a field in DeliveryAddress that has several parts of an address that are separated by ';' (g_AddressValues).  In the test data, there are multiple ';' in a row, indicating that a particular member of g_AddressValues is being skipped.  With 'anyEmpty' this fails to parse, but with 'trailingEmpty' it parses as expected.  I'm not sure if this is potentially a bug in Daffodil's separator suppression code or if this should have been using trailingEmpty all along.